src/plugins/circular_view/src/CircularView.cpp

```diff
@@ -1126,7 +1126,7 @@ qint64 CircularViewRenderArea::asinToPos(const qreal asin) const {
         graduatedAngle += 360 * CircularView::GRADUATION;
     }
     qint64 seqLength = circularView->getSequenceLength();
-    qint64 resultPosition = (seqLength * graduatedAngle) / CircularView::MAX_GRADUATION_ANGLE + 0.5f;
+    qint64 resultPosition = (seqLength * graduatedAngle) / CircularView::MAX_GRADUATION_ANGLE;
```

Comment:
Always use an explicit type conversion (qRound/qFloor) to avoid the errors like you fixed above

Reply:
OK